scripts/recovery/disaster_recovery_drill.sh

```
@@ -0,0 +1,434 @@
+    else
+        log "WARN" "성능 테스트 주의 - 쿼리 시간: $query_time"
+        return 1
+    fi
+}
+
+# 복구 리허설 리포트 생성
+generate_drill_report() {
+    log "INFO" "복구 리허설 리포트 생성"
+    
+    local report_file="$DRILL_DIR/drill_report_${DATE_SUFFIX}.json"
+    local total_duration=$((ttr_metrics["total_duration"]:-0))
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)